props/saber_fett263_buttons.h

```diff
@@ -5832,6 +5897,7 @@ SaberFett263Buttons() : PropBase() {}
   bool auto_lockup_on_ = false; // Battle Mode Lockup active
   bool auto_melt_on_ = false; // Battle Mode Melt/Drag active
   bool battle_mode_ = false; // Battle Mode active
+  bool preon_effect_ = false; // used for interactive preon
```

Comment:
This variable (and it's comment) are confusing.
What is it for?
Why set it in SB_Effect()?
When is it reset back to false?

Reply:
comment updated, for Interactive Preon to work properly EFFECT_INTERACTIVE_PREON sets to true, then the conditional effect for EFFECT_FAST_ON within the style resets to false.  This is to allow the "interactive" part as much time as the user wants based on the style code.  Calling EFFECT_FAST_ON from within the Interactive Preon is how we ignite the blade based on the condition set within the effect.